portfolio/src/main/webapp/quiz.js

```diff
@@ -2,40 +2,91 @@
  * Question data type.
  */
 class Question {
-  constructor(question, answers, correct) {
+  constructor(question, answers, correctAnswerIndex) {
     this.question_ = question;
     this.answers_ = answers;
     this.correctAnswerIndex_ = correctAnswerIndex;
   }
 
   getQuestion() {
-    return question_;
+    return this.question_;
   }
 
   getAnswers() {
-    return answers_;
+    return this.answers_;
   }
 
   getCorrectAnswerIndex() {
-    return correctAnswerIndex_;
+    return this.correctAnswerIndex_;
   }
 }
 
+// create array of quiz questions 
+const questions = [];
+questions.push(
+    new Question('What\'s my GitHub username?',
+		['iolawale', 'ihsan', 'olawale314', 'ihsan314'], 3));
+questions.push(
+    new Question('Where do I reside?',
+		['San Francisco, CA', 'Vancouver, BC', 'Toronto, ON', 'Palo Alto CA'], 1));
+questions.push(
+    new Question('Where do I attend university?',
+		['University of British Columbia', 'Simon Fraser University',
+		  'Stanford University', 'New York University'], 0));
+questions.push(
+    new Question('In which of these classes did I receive the highest grade?',
+		['Principles of Microeconomics', 'Honours Integral Calculus',
+		  'Basic Algorithms and Data Structures', 'Introduction to Microcomputers'], 1));
+questions.push(
+    new Question('In which of these classes did I receive the lowest grade?',
+		['Principles of Microeconomics', 'Honours Integral Calculus',
+		  'Basic Algorithms and Data Structures', 'Introduction to Microcomputers'], 3));
+
 function presentQuiz() {
-  // create array of quiz questions 
-  const questions;
+  const questionsContainer = document.getElementById('questions-container');
+  for (let questionNum = 0; questionNum < questions.length; questionNum++) {
+    const question = questions[questionNum];
+    const answers = question.getAnswers();
+    const correctIndex = question.getCorrectAnswerIndex();
+    // Display the options
+    const questionStatementElement = document.createElement('p');
+    const questionStatementText = document.createTextNode(`Question ${questionNum}: ${q.getQuestion()}`);
```

Comment:
Should q be question instead?

Reply:
Yes that is the case, I will resolve that.